scripts/ruby/lib/file_types/file_csv.rb

```diff
@@ -26,7 +37,17 @@ def row_to_solr(doc, headers, row)
   end
 
   def transform_es
-    raise "Currently CSV to ES transformation is not supported"
+    puts "transforming #{self.filename}"
+    es_doc = []
+    @csv.each do |row|
+      if !row.header_row?
+        es_doc << row_to_es(@csv.headers, row)
+      end
+    end
+    if @options["output"]
+      filepath = "#{out_es}/#{self.filename(false)}.xml"
```

Comment:
I know I haven't been consistent across other code reviews, but I remembered it this time. We should be using that `join` method for file paths for portability. Maybe just make an issue to tackle that separately from this PR? Might even be worth having a general issue to look through a variety of our repos :grimacing:

Reply:
Good idea, it wouldn't be hard to change that for this particular setting because the majority of the output stuff is in `file_type.rb` so it would be easy to hit them all at once.

Reply:
Awesome! Yay for easy changes.